README.md

```diff
@@ -396,6 +399,23 @@ subclass will be returned from all SDK methods that previously returned
 an instance of the parent. See `BaseAPIJSONObjectMeta` and `Translator`
 to see how the SDK performs dynamic lookups to determine return types.
 
+# Versions
+We use a modified version of [Semantic Versioning](https://semver.org/) for all changes. See [version strategy](VERSIONS.md) for details which is effective from 30 July 2022.
+
+## Supported Version
+
+Only the current MAJOR version of SDK is supported. New features, functionality, bug fixes, and security updates will only be added to the current MAJOR version.
+
+A current release is on the leading edge of our SDK development, and is intended for customers who are in active development and want the latest and greatest features.  Instead of stating a release date for a new feature, we set a minor or path release cadence of 2-3 months. At the same time, there is no schedule for major or breaking release. Instead, we will communicate one quarter in advance the upcoming breaking change to allow customers to plan for the upgrade. We always recommend that all users run the latest available minor release for whatever major version is in use. We highly recommend upgrading to the latest SDK major release at the earliest convenient time and before the EOL date.
```

Comment:
Are we sure if we want to specify cadence? 2-3 months is too rarely IMO. Maybe this time will be changing so what about sth like this:
`a minor or patch fixed release cadence`

Reply:
didn't thought about that 2-3 month is fixed and we can't release it frequently. It was rather to say that at least every 2-3 months we'll push new version.